### ISSUE TITLE
Rapid Router Student dashboard - Wireframes

### ISSUE THREAD

**Issue by j4mesholland** [ocadotechnology/rapid-router#968]
## Description ##
Create wireframes to update the student dashboard (play/details/)

## Acceptance criteria ## 
- [x] To use the current portal design style for consistency
- [x] Use the bootstrap 12 column grid for layouts (see existing Portal Sketch files)
- [x] Make use of the sticky sub-nav bar or menus for consistency to replace the current sidebar
- [x] Implement layout changes to allow space for AI:MMO and future games
- [x] Decide if we need all the current content (EG: Rapid Router characters)

## Metrics ##
- [ ] Create prototypes for user testing: https://app.zenhub.com/workspace/o/ocadotechnology/rapid-router/issues/969

**Comment by mariaFernando** [comment]
"AI:MMO is a Massively Multiplayer Online (MMO) AI learning tool"
- maybe rephrase this? "AI learning tool" might be confusing, could we just put "learning tool"? Since it's mentioned later that it teaches AI
@j4mesholland

**Comment by j4mesholland** [comment]
Sure, lets just put 'learning tool'

**Comment by j4mesholland** [comment]
Lets add the designs here...

**Comment by j4mesholland** [comment]
@mariaFernando Can you add any revisions to your version, or have you finished?

**Comment by mariaFernando** [comment]
I'll add a few
